src/levels.rs

```diff
@@ -1,13 +1,16 @@
 use std::ffi::c_uint;
 use std::ops::Neg;
-use strum::EnumCount;
-
-pub type ObuMetaType = c_uint;
-pub const OBU_META_TIMECODE: ObuMetaType = 5;
-pub const OBU_META_ITUT_T35: ObuMetaType = 4;
-pub const OBU_META_SCALABILITY: ObuMetaType = 3;
-pub const OBU_META_HDR_MDCV: ObuMetaType = 2;
-pub const OBU_META_HDR_CLL: ObuMetaType = 1;
+use strum::{EnumCount, FromRepr};
+
+#[repr(u32)]
```

Comment:
Why do we need this?  Shouldn't the below code still work fine if it defaults to `u8`?

Reply:
generally yes, but it's slightly less convenient. To convert from `u32` to `ObuMetaType` you'd now need to go via `u8`, something like

```rust
impl TryFrom<c_uint> for ObuMetaType {
    type Error = Rav1dError;

    fn try_from(value: c_uint) -> Result<Self, Self::Error> {
        let value = u8::try_from(value).map_err(|_| Rav1dError::EINVAL)?;
        Self::from_repr(value).ok_or(Rav1dError::EINVAL)
    }
}
```

that's just more code for no real gain in my view, but happy to go with that solution.